### PR TITLE
Release announcement: 3.0

### PR DESCRIPTION
**Pull request by real34**
This PR adds the 3.0 announcement to our changelog.

**[Preview](https://deploy-preview-773--heuristic-almeida-1a1f35.netlify.app/changelog/front-commerce-3.0)**

### CODE REVIEW

File: changelog/front-commerce-3.0.mdx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+**We could achieve this thanks to a major shift in the way data flows in
+Front-Commerce**: instead of using GraphQL over HTTP (2.x), Front-Commerce now
+uses GraphQL server-side and uses Remix's default fetching patterns (`loaders` /
+`actions` / `fetchers`) for client-server interactions. Read
+[Server vs. Client Code Execution (Remix doc)](https://remix.run/docs/en/main/discussion/server-vs-client)
+to learn about this.
+
+### Streaming Responses to Users
+
+This new version, improves how content is delivered to your users. Unlike
```
Comment (Kagaminara) [inline changelog/front-commerce-3.0.mdx:78]: **thought:** this comma feels a bit weird here, splitting the sentence that is already not that long.

```suggestion
This new version improves how content is delivered to your users. Unlike
```

**Review by mGGk-fr — APPROVED** [review]

**Review by Kagaminara — APPROVED** [review]
Let's go ! 🎉

File: changelog/front-commerce-3.0.mdx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+patience as we focused on v3 development over summer features. We're eagerly
+anticipating how this new release will enhance your business.
+
+Together, we've created something special: let's dive into what this release
+holds and how it will positively impact all of you. ☕
+
+## Web Performance Improvement
+
+We're all working together to create user experiences that convert, fast! Let's
+take a moment to explore the improvements that will led to faster page load
```
Comment (paul-vd) [inline changelog/front-commerce-3.0.mdx:55]:
```suggestion
take a moment to explore the improvements that will lead to faster page load
```

File: changelog/front-commerce-3.0.mdx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+development workflows might not be very clear at a first glance, but we're
+excited to incorporate this into your experience around Front-Commerce … but
+that's another story.
+
+## Welcome to v3
+
+If you're still reading, you must have plenty of questions about the next steps
+for you.
+
+We can't address anything in one blog post, but we're happy to help you out!
```
Comment (paul-vd) [inline changelog/front-commerce-3.0.mdx:205]:
```suggestion
We can't address everything in one blog post, but we're happy to help you out!
```

File: changelog/front-commerce-3.0.mdx [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+Some highlights:
+
+- in the next weeks, we'll keep publishing concrete content to zoom on new
```
Comment (paul-vd) [inline changelog/front-commerce-3.0.mdx:209]:
```suggestion
- in the next weeks, we'll keep publishing concrete content to zoom in on new
```

File: changelog/front-commerce-3.0.mdx [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+  Front-Commerce concepts so you'll keep learning more about this. Follow us!
+- yes, we recommend that new Front-Commerce projects start with 3.0
```
Comment (paul-vd) [inline changelog/front-commerce-3.0.mdx:211]: I would remove the `yes` here as no question has been asked, unless reformatting this section into a Q&A or FAQ

```suggestion
- We recommend that new Front-Commerce projects start with 3.0
```

File: changelog/front-commerce-3.0.mdx [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+- migrating a 2.x project to 3.x is not only possible but we expect you to start
+  planning this transition. Our `@front-commerce/compat` package is dedicated to
+  this use case, so you can start migrating right away and refactor your
+  codebase progressively. The best thing is to <ContactLink>discuss it with
+  us</ContactLink>. For information, the first project starts its migration on
+  Monday!
+- we will continue to ship new versions of Front-Commerce 2.x, but don't expect
+  major features on this branch by default
+- yes, we still expect some performance improvements and stabilization work in
```
Comment (paul-vd) [inline changelog/front-commerce-3.0.mdx:220]: Same as previous suggestion remove `yes`

```suggestion
- We still expect some performance improvements and stabilization work in
```

**Review by paul-vd — APPROVED** [review]
Some minor nitpicking

**Pull request merged**